docs/README.md

```diff
@@ -6,7 +6,7 @@ What's this?
 
 This directory contains the source code for Mypy documentation (under `source/`)
 and build scripts. The documentation uses Sphinx and reStructuredText. We use
-`sphinx-rtd-theme` as the documentation theme.
```

Comment:
I think it would be easier for you to recreate your fork + PR.

You have lots of unrelated changes in this one 🙂